azure-mgmt-containerservice/pom.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>azure-mgmt-containerservice</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Microsoft Azure SDK for Container Service Management</name>
+    <description>This package contains Microsoft Azure Container Service SDK.</description>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+    <licenses>
+        <license>
+            <name>The MIT License (MIT)</name>
+            <url>http://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
```

Comment:
why use SDK repo

Reply:
I'm not sure about it, all the poms are used it in this project. Meanwhile, since we will migrate back to sdk, may it be ok?